src/context.cpp

```diff
@@ -257,9 +294,14 @@ MENU_CONTROL Context::spray(
 
 MENU_CONTROL Context::set_desired_soil_moisture_to_current()
 {
-    // Set our desired soil moisture to match the last known soil moisture
+    // Set our desired soil moisture to match the last known soil moisture, in memory and NVS
     CONTEXT_LOCK(/* RET_VAL = */ MENU_CONTROL_RELEASE);
     desired_soil_moisture = current_soil_moisture;
+    (void) storage_get(
```

Comment:
Wrong, use `storage_set`.

Reply:
Addressed in https://github.com/jCallon/squirt/pull/3/commits/09245cdc3c711cc6a98ca41015751c44460d8aab.